gui/support/test-helpers/clickNextPage.js

```diff
@@ -1,4 +1,5 @@
 export default wrapper => {
+  wrapper = wrapper.update() // Fixes enzyme finder bug
```

Comment:
Got a link to this bug / issue on github / on stack overflow? What is the bug?

Reply:
https://github.com/airbnb/enzyme/issues/1233#issuecomment-385343903

Reply:
This should be in the code ;)